### PR TITLE
Add run-rp docs

### PR DESCRIPTION
**Pull request by SudoBrendan**
### Which issue this PR addresses:

Follow up to split between `runlocal-rp` and `run-rp` - these docs outline how to use the new feature of containerized builds and RP execution

- adds documentation on build/run dependencies of containerized build
- adds instructions on how to use 'make run-rp'

### What this PR does / why we need it:

Docs

### Test plan for issue:

N/A - docs

### Is there any documentation that needs to be updated for this PR?

Yes, this PR

### How do you know this will function as expected in production? 

N/A - docs

### CODE REVIEW

**Review by shubhadapaithankar — APPROVED** [review]

File: docs/deploy-development-rp.md [hunk 1]
```diff
@@ -104,9 +104,19 @@ mock a cluster MSI. This script will also create the platform identities, platfo
...
-   ```
+    Option 1: using local compilation and binaries (requires local `go`/build dependencies/etc):
+    ```bash
+    make runlocal-rp
+    ```
+
+    Option 2: using containerized build and run (requires local `podman` and `openvpn`):
+    ```bash
+    # establish a VPN connection to the shared dev environment Hive cluster
+    sudo openvpn secrets/vpn-${LOCATION}.ovpn &
```
Comment (ehvs) [inline docs/deploy-development-rp.md:115]: From my testing it must be `vpn-aks-$LOCATION.ovpn`

**Review by ehvs — COMMENTED** [review]
Perhaps the containerized mode could use either its own page or its own sessions on how setup it, with a shorter set of commands that is required only to succeed `make run-rp`.

File: docs/deploy-development-rp.md (hunk 1 above)
Comment (SudoBrendan) [reply docs/deploy-development-rp.md:115]: There should be variations of both `secrets/vpn-${LOCATION}.ovpn` and `secrets/vpn-aks-${LOCATION}.ovpn` for every dev region. I've always used the non-aks one locally :thinking: and have tested this in `westeurope` with success... do we have diverging dev environments maybe?

Comment (hlipsig) [reply docs/deploy-development-rp.md:115]: @ehvs Can you follow up on this documentation task in a new PR? Maybe we can reconcile the difference, but I don't think we should continue to block the PR.

**Review by hlipsig — APPROVED** [review]

**Pull request merged**